app/controllers/api/v0/admin/configs_controller.rb

```diff
@@ -0,0 +1,26 @@
+module Api
+  module V0
+    module Admin
+      class ConfigsController < ApiController
+        before_action :authorize_admin
+        after_action :verify_authorized
+
+        respond_to :json
```

Comment:
You can move this into `routes.rb` which would be more consistent with how  we do it elsewhere:

```ruby
resources :configs, only: %i[show create], defaults: { format: :json }
```